packages/scene-composer/src/store/slices/ViewOptionStateSlice.ts

```diff
@@ -10,10 +10,12 @@ export interface IViewOptionStateSlice {
   }>;
   tagSettings?: ITagSettings;
   enableMatterportViewer?: boolean;
+  defaultLighting?: string;
```

Comment:
Is this truly a string that can be anything or should it be a literal string union (e.g., ‘neutral’ | ‘something’ | undefined) with a defined set of values?

Reply:
If the parameter is passed, it will be a `string`.

Reply:
... aaaand that just sank in. I can set it to use an `enum`.

Reply:
Fixed!